tests/data/attributes/operator/test_operator.py

```diff
@@ -174,17 +174,20 @@ def test_value_init(self, attribute_cls, op_in):
         """Test that a DatasetOperator can be value-initialized
         from an operator, and that the deserialized operator
         is equivalent."""
-        if not qml.operation.active_new_opmath() and isinstance(op_in, qml.ops.LinearCombination):
-            op_in = qml.operation.convert_to_legacy_H(op_in)
+        with np.printoptions(legacy="1.21"):
```

Comment:
Do we want 1.21 or 1.25 here?
I'd also argue this needs to have a ticket created (to be handled another time) for updating the datasets to the new syntax @albi3ro @Alex-Preciado

Reply:
The problem with using 1.25 is what I mentioned in the slack channel, because in some action script gets downgraded to scipy < 1.13, which then downgrades numpy to < 2.0, such that 1.25 is not available as an option anymore.